display-guild.lic

```diff
@@ -0,0 +1,91 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#display-guild
+=end
+
+no_pause_all
+no_kill_all
+
+arg_definitions = [
+    [
+	{ name: 'debug', regex: /debug/i, optional: true, description: 'Very verbose, may cause lich to hang for several seconds if using large contact files' }
+	]
+]
+args = parse_args(arg_definitions)
+$debug_mode_cc = args.debug
+
+Contacts = YAML.load_file("./scripts/data/base-contacts.yaml")
+Contacts = Contacts['contacts']
+
+#DEBUG INFO
+echo "---Base Contacts---" if $debug_mode_cc
+echo Contacts.inspect if $debug_mode_cc
+echo Contacts.class if $debug_mode_cc
+
+#Check for <CharName>-contacts.yaml file and merges it if exists
+if File.exist?("./scripts/profiles/#{checkname}-contacts.yaml")
+	echo "Personal contact file found - now importing." if $debug_mode_cc
+	
+	Personal_Contacts = YAML.load_file("./scripts/profiles/#{checkname}-contacts.yaml")

```

Comment:
Hmmm, that's a great point that I hadn't really thought about (I only have one character myself). I feel like people would definitely want that information shared across characters, and duplicating the file and updates would be a bit of a pain.

Maybe the best bet would be to have it referenced as "personal-contacts.yaml" still out of the /scripts/profiles directory, but also put it in the gitignore so that other contributors don't accidentally upload theirs and overwrite it. It would still be the same amount of work as copying the sample file to yourname-contacts is now, and is a process users are already used to with the setup yamls.

I had also had the thought that contacts could just be added to the setup yaml (and tried to structure the yaml such that it would be an easy conversion) but I'm worried that for some users who have several dozen or hundreds of contacts, it would make their setup yaml too messy.